### ISSUE TITLE
postinstall script for `@napi-rs/canvas` uses musl instead of glibc

### ISSUE THREAD

**Issue by theakman2** [oven-sh/bun#7617]
### What version of Bun is running?

1.0.17+5e60861c3

### What platform is your computer?

Linux 6.5.0-14-generic x86_64 x86_64

### What steps can reproduce the bug?

1. Create a directory X.
2. Create file at `X/package.json` with the following content:

```json
{
	"dependencies": {
		"@napi-rs/canvas": "~0.1.44"
	}
}
```

3. Run `bun i` from directory X.
4. Observe that installation fails with exit code 1.

### What is the expected behavior?

`bun i` runs successfully.

### What do you see instead?

```
a@a:/tmp/repro$ bun i
bun install v1.0.17 (5e60861c)

error: install script from "@napi-rs/canvas-linux-x64-musl" exited with code 1 (SIGHUP)
a@a:/tmp/repro$ echo $?
1
```

### Additional information

- I'm running Xubuntu 23.10.
- Deleting the `bun.lockb` and `node_modules` directories before running `bun i` doesn't work.
- `bun i` worked in v1.0.16 (although this may just be because the install scripts weren't executed before 1.0.17). This is a regression in v1.0.17.
- `npm i` works (tested with Node 18 and Node 20).

**Comment by CeebDev** [comment]
Any ETA for this ?

**Comment by n1ckoates** [comment]
is this issue resolved? cannot reproduce on latest version of Bun.

**Comment by Jarred-Sumner** [comment]
Yes, it was fixed in Bun v1.0.23
